src/components/Attachment/Card.tsx

```diff
@@ -67,12 +71,38 @@ const goToURL = (url) => {
   });
 };
 
+export type CardProps<
+  At extends UnknownType = DefaultAttachmentType
+> = Attachment<At> & {
+  /**
+   * Position of the message, either 'right' or 'left'
+   */
+  alignment: 'right' | 'left';
```

Comment:
we have this type within MessagesContext

Reply:
gotcha, will replace with the Alignment type 👍🏻